src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -335,6 +339,7 @@ final class HomeViewController: UIViewController {
 		collectionView.translatesAutoresizingMaskIntoConstraints = false
 		collectionView.isAccessibilityElement = false
 		collectionView.shouldGroupAccessibilityChildren = true
+		collectionView.alwaysBounceVertical = true
```

Comment:
hm, why?

Reply:
feels nicer in my opinion, otherwise the app feels very static when there are few cells on the screen.. but this is just my personal preference :)